### PR TITLE
resync mixs crd all

### PR DESCRIPTION
**Pull request by jmuk**
- kubernetesenv -> kubernetes, renamed by #2543
- reportnothing: seems duplicated

### CODE REVIEW

File: install/kubernetes/templates/istio-mixer.yaml.tmpl [hunk 1]
```diff
@@ -554,23 +554,6 @@ spec:
   scope: Namespaced
   version: v1alpha2
 ---
-
-kind: CustomResourceDefinition
-apiVersion: apiextensions.k8s.io/v1beta1
-metadata:
-  name: reportnothings.config.istio.io
```
Comment (douglas-reid) [inline install/kubernetes/templates/istio-mixer.yaml.tmpl]: curious: why is reportnothings disappearing?

Comment (jmuk) [reply install/kubernetes/templates/istio-mixer.yaml.tmpl]: Yeah, expand the unedited parts a few times, you'll see another reportnothings.

**Comment by jeffmendoza** [comment]
FYI I merged this locally and deployed to a cluster. All my metrics have valid attributes again! Thanks :+1:

**Comment by mandarjog** [comment]
Thanks @jeffmendoza

**Pull request merged**